standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -74,7 +74,7 @@ public LocalPravegaEmulator build() {
                     .enableRestServer(enableRestServer)
                     .enableAuth(enableAuth)
                     .enableTls(enableTls)
-                    .tlsProtocolVersion(tlsProtocolVersion)
+                    //.tlsProtocolVersion(tlsProtocolVersion)
```

Comment:
Shouldn't both of these facilities still accept overrides instead of only being hardcoded? In other words, sane defaults should be used unless `.tlsProtocolVersion()` is called to override the defaults.

Reply:
Yes. Added default values in both LocalPravegaEmulator and InProcPravegaCluster classes. They are used unless `.tlsProtocolVersion()` is called to override them.